### PR TITLE
Add create escrow form validation

### PR DESCRIPTION
**Pull request by jrainville**
image

### CODE REVIEW

**Comment by jrainville** [comment]
Rebased and added the library you suggested.
Only thing, I needed to keep the expiration manual validation because it is not an input but a custom component so the validator does not work on it

File: app/js/components/EscrowList.jsx [hunk 1]
```diff
@@ -67,7 +68,7 @@ const EscrowList = (props) => (
             <td>{getEscrowStateText(escrow, props.t)}</td>
             <td>{escrow.buyer === web3.eth.defaultAccount ? escrow.seller : escrow.buyer}</td>
             <td>{escrow.amount}</td>
-            <td>{new Date(escrow.expirationTime * 1000).toString()}</td>
+            <td>{moment(escrow.expirationTime * 1000).toString()}</td>
```
Comment (alaibe) [inline app/js/components/EscrowList.jsx]: I can see we are doing this multiple times, do you think we should move this conversion to the selector?

File: app/js/features/escrow/reducer.js [hunk 2]
```diff
@@ -40,6 +40,11 @@ const escrowBuilder = function (escrowObject) {
 
 function reducer(state = DEFAULT_STATE, action) {
   let escrows  = cloneDeep(state.escrows);
+  let currentEscrow;
+  if (action.escrowId) {
+    currentEscrow = escrows.find(escrow => escrow.escrowId === action.escrowId);
+  }
+
```
Comment (alaibe) [inline app/js/features/escrow/reducer.js]: What if currentEscrow is undefined at this stage (line 47)?
Is it even possible?

Comment (jrainville) [reply app/js/features/escrow/reducer.js]: It's possible that escrowId is available but it's a new escrow (eg creating a new escrow), but the reducer case doesn't use `currentEscrow` so it's not an issue there. 
We could add a condition on the cases that require `currentEscrow` and throw an error, but that's what's gonna happen anyway.

**Comment by jrainville** [comment]
@alaibe Fixed and rebased

**Pull request merged**